src/function_aliases.R

```diff
@@ -0,0 +1,30 @@
+#complete this list with any common, globally (across macrosheds) used funcs.
+#refer to everything else explicity with ::
+#this will make packaging easier and more professional
+
+read_csv = readr::read_csv
```

Comment:
Genuine question, and I think you are right, but = here is intentional (over <-?) maybe you can explain someday.

Reply:
here, specifically? I use ' = ' for variable assignment everywhere just because it takes up one fewer character, is unambiguous, and is the assignment operator for every other language in the universe. I differentiate it from parameter assignment with spaces. This is all part of a style conversation we should have with Spencer when he starts though, because there are gray areas. Like in mutate(), I've started using " = " instead of "=", because that feels more like variable assignment than parameter assignment, and it improves readability.